processor/attributes/testdata/config.yaml

```
@@ -0,0 +1,131 @@
+processors:
+  # The following example demonstrates the default behavior of insert a key/value.
+  # There is no need to specify action because the default action is insert.
+  attributes/default:
+    actions:
+      # The following inserting a new attribute {"attribute1": 123} to spans where
```

```suggestion
      # The following is inserting a new attribute {"attribute1": 123} to spans where
```
or `...will insert...`